src/main/java/com/kodedu/controller/ApplicationController.java

```diff
@@ -87,6 +87,7 @@
 import javax.imageio.ImageIO;
 import javax.imageio.ImageReader;
 import javax.imageio.stream.ImageInputStream;
+import java.awt.geom.IllegalPathStateException;
```

Comment:
Can we use java.lang. IllegalStateException instead of java.awt.geom.IllegalPathStateException ? Since we don't prefer to use any component under java.awt.

Reply:
Done